provider/cluster_rosa_classic_resource.go

```diff
@@ -349,6 +350,15 @@ func (t *ClusterRosaClassicResourceType) GetSchema(ctx context.Context) (result
 					ValueCannotBeChangedModifier(t.logger),
 				},
 			},
+			"channel_group": {
+				Description: "Name of the channel group from which to select the OpenShift cluster version, for example 'stable'.",
+				Type:        types.StringType,
+				Optional:    true,
+				Computed:    true,
+				PlanModifiers: []tfsdk.AttributePlanModifier{
+					ValueCannotBeChangedModifier(t.logger),
```

Comment:
There are ways to change the clusters channel group btw, I have never done it though

Reply:
Should I allow it to be modified?

Reply:
I think better not to support it...